_projects/0100_medicare-payment-program.md

```diff
@@ -18,24 +18,29 @@ project_url:
 
 # Impact statement (Edit this)
 impact_statement:
-  - figure: "57"
+  - figure: "53"
     unit: "M"
     description: |-
       Americans depend on Medicare payments for their healthcare
 
-  - figure: 10
+  - figure: 65
     unit: "K"
     description: |-
-      Additional people enroll in Medicare per day
+      New monthly claims priced in the cloud‑based service during the first few months of 2020
+
+  - figure: >75
```

Comment:
```suggestion
  - figure: ">75"
```
I suspect the build process is getting confused about the `>` symbol - this happens when I try a build:
> Error: YAML Exception reading /[...]/_projects/0100_medicare-payment-program.md: (<unknown>): did not find expected comment or line break while scanning a block scalar at line 31 column 13

Reply:
it might be that this error is preventing the file from being used for the Projects page.
Preview: https://fish-responsive-otter-gi.app.cloud.gov/projects